enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/QueryStatusResult.java

```diff
@@ -43,10 +43,22 @@
     public final Map<String,Object> parameters;
     public final String startTime;
     public final String elapsedTime;
+    /** EXPERIMENTAL: added in Neo4j 3.2 */
```

Comment:
Do these comments add some kind of value? Shouldn't the experimental nature be properly documented elsewhere? Somewhere user facing that is. Git blame can already inform an interested dev that it was added in 3.2.